### PR TITLE
Disable fork button for view_only user

### PR DESCRIPTION
**Pull request by tonyjiangh**
Fix https://github.com/getredash/redash/issues/2459

### Disabled view
image
With cursor disabled as well

### ~~Before/After~~
Permission | Before (Show Data Only) | After (Show Data Only) | Before(Edit Source) | After(Edit Source)
---|---|---|---|---
Creator/Admin | Active | Disabled | Active  | Active |
Full Access | Active | Disabled | Active | Active | 
View Only | Active(returns 403 response) | Disabled | Active(returns 403 response) | Disabled | 

Tested over Firefox 59.0.2

### CODE REVIEW

File: client/app/assets/less/inc/dropdown.less [hunk 1]
```diff
@@ -29,10 +29,12 @@
 
     &:not([class*="bg-"]) {
         & > li > a {
-            color: #4C4C4C;
```
Comment (kravets-levko) [inline client/app/assets/less/inc/dropdown.less]: @tonyjiangh I think, [this is a better place](https://github.com/getredash/redash/blob/master/client/app/assets/less/inc/dropdown.less#L9-L15) for this CSS. @kocsmy - WDYT?

**Comment by arikfr** [comment]
Thanks! Why disable it (for creator/admin) in data only mode?

**Comment by kravets-levko** [comment]
@arikfr As I see, @tonyjiangh didn't implement any new logic - he uses the same condition (`query.id === undefined || !canForkQuery()`), so `Fork` item will not be disabled in data only mode (as it works now).

File: client/app/assets/less/inc/dropdown.less (hunk 1 above)
Comment (kocsmy) [reply client/app/assets/less/inc/dropdown.less]: Yes, we have a disabled class for li already. We'll need to adjust the color a bit more gray to indicate it's disabled.

**Comment by tonyjiangh** [comment]
@kravets-levko 
Thanks for the review. I'll try to fix the position for css.

@arikfr @kravets-levko 
Yes, I didn't implemented any new logic. But fork in data only mode would be disabled with this fix.
Here's why:
We're using `client/app/pages/queries/view.js` for data only view,
and `client/app/pages/queries/source-view.js` for query source view.
`$scope.canForkQuery()` is only currently implemented for the later.

It thought this was intentional. But based on the conversation above, I guess it's not. I'll fix that too.

**Comment by arikfr** [comment]
Thanks!

**Pull request merged**